packages/blocto/src/index.ts

```diff
@@ -28,9 +28,7 @@ function BloctoWallet(): WalletInit {
               return item.id === (params && params[0] && params[0].chainId)
             })
             if (!chain) throw new Error('chain must be set before switching')
-            const providerRpcurl =
-              bloctoProvider.switchableNetwork[chain.id] &&
-              bloctoProvider.switchableNetwork[chain.id].rpc_url
+            const providerRpcurl = bloctoProvider.rpc && bloctoProvider.rpc
```

Comment:
Are both of these needed?

Reply:
fixed in this commit [a065226](https://github.com/blocknative/web3-onboard/pull/2082/commits/a065226661afe3a67d91c13193171ba249611675) , thx